manifests/eap/module.pp

```diff
@@ -26,6 +26,7 @@
     $client_cert_source,
     $metadata_roles=undef,
 )
+inherits lightblue::eap
```

Comment:
Inheritance is kind of frowned upon in puppet. What variables are having issues?